README.md

```diff
@@ -11,3 +11,47 @@ Add following step to your workflow:
     delete_release: true #(optional) default: true 
     repo: <owner>/<repoName> #(optional) target repository. default: repo running this action
 ```
+
+## Developing
+
+You can follow these instructions to get this project cloned to your local machine and ready for development. Note that
+development requires at least [Node 16](https://nodejs.org/en/download) to be installed on your computer.
+
+```shell
+git clone https://github.com/dev-drprasad/delete-tag-and-release.git
+cd delete-tag-and-release
+npm install
+```
+
+Tests can be run via:
+
+```shell
+npm test
+```
+
+Since this is a repo for a GitHub Action, **every new commit that changes code must include a new bundle.** You can 
```

Comment:
```suggestion
Since this is a repo for a [GitHub Action](https://docs.github.com/en/actions), **every new commit that changes code must include a new bundle.** You can 
```